src/test/java/uk/gov/companieshouse/companyprofile/delta/mapper/CompanyProfileEnumMapperTest.java

```diff
@@ -0,0 +1,367 @@
+package uk.gov.companieshouse.companyprofile.delta.mapper;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.util.FileCopyUtils;
+import uk.gov.companieshouse.api.company.CompanyProfile;
+import uk.gov.companieshouse.api.company.Data;
+import uk.gov.companieshouse.api.delta.BooleanFlag;
+import uk.gov.companieshouse.api.delta.CompanyDelta;
+
+import java.io.InputStreamReader;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+@ExtendWith({SpringExtension.class, MockitoExtension.class})
+@ContextConfiguration(classes = {CompanyProfileEnumMapperImpl.class})
+public class CompanyProfileEnumMapperTest {
+
+    private ObjectMapper mapper;
+
+    private CompanyDelta companyDelta;
+
+    private CompanyProfile companyProfile;
+
+    private Data expectedOutputData;
+
+    @Autowired
+    CompanyProfileEnumMapper companyProfileEnumMapper;
+
```

Comment:
These tests are covered in the CompanyMapperTest now so we should be able to remove this test I think?